README.md

```diff
@@ -33,19 +33,15 @@ To setup the project locally read these wiki pages and follow the instructions:
 
 ### Run app
 
-The project runs on Python 3.
+The project runs on Python 3. We use [Poetry](https://python-poetry.org/) to manage dependencies.
```

Comment:
do you have any instructions you could put on how to upgrade dependencies and downgrade, add a new dependency to the project, etc? that would be great

Reply:
That's a good idea. Added it:)